feature-configs/base/sriov/06-app-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
```

Comment:
I would remove this whole file and have the namespace as part of the kustomizatoin of the network.